Sources/Passage/AuthorizationController/PassageAutofillAuthorizationController.swift

```diff
@@ -45,12 +45,13 @@ public class PassageAutofillAuthorizationController : NSObject, ASAuthorizationC
         }
     }
     
-    public func begin(onSuccess:  ((AuthResult) -> Void)?, onError: ((Error) -> Void)?, onCancel: (() -> Void)? ) async throws -> Void {
+    public func begin(anchor: ASPresentationAnchor, onSuccess:  ((AuthResult) -> Void)?, onError: ((Error) -> Void)?, onCancel: (() -> Void)? ) async throws -> Void {
```

Comment:
I originally removed this (and thought removing it was fine) because a customer using this package with SwiftUI complained about the fact that this shouldn't be necessary, or possible in swift UI. 

https://passage-identity.atlassian.net/browse/PSG-1484?search_id=70563e91-f60a-497d-8cba-8b1c80f9d044

Reply:
@macleanevans do you remember where the specific customer feedback was?

Reply:
Gotcha. Yeah, we may need to have a separate method (or even a separate class) for SwiftUI-autofill use case, but this anchor is still needed for now.

Reply:
Cool, sounds like that work needs to get split out and investigated a bit further then

Reply:
Yeah, I believe @macleanevans mentioned needing to add more wholly and explicitly add SwiftUI support someday soon.

Reply:
@rickycpadilla Can you help me understanding why our SDK methods need SwiftUI or Storyboard functions/bindings? That seems like something that should be a separate concern (but I'm a mobile native newb).